app/models/post.rb

```diff
@@ -1,3 +1,16 @@
 class Post < ApplicationRecord
-  belongs_to :author
+  after_save :update_posts_counter
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def recent_five
```

Comment:
- [ ] [OPTIONAL] I would like to suggest that you give your methods names that can be easily understood by other developers going through your codebase. It will be nice to give this method
``` ruby
def recent_comments
```